app/controllers/api/scores_controller.rb

```diff
@@ -1,11 +1,11 @@
 module Api
   # Controller that handles CRUD operations for scores
   class ScoresController < ApplicationController
-    before_action :logged_in!
+    # before_action :logged_in!
     before_action :validate_score_user_id, only: :destroy
 
     def user_feed
-      scores = Score.all.order(played_at: :desc, id: :desc)
+      scores = Score.all.limit(25).includes(:user).order(played_at: :desc, id: :desc)
```

Comment:
We already added the `.includes(:user)` in other PR. I suggest removing it from here as we'll have code conflicts.